test/test.js

```diff
@@ -328,6 +328,36 @@ describe('express-winston', function () {
         loggerFn.should.throw();
       });
     });
+
+    describe('skip option', function() {
+      it('should log error by default', function() {
+        var options = {
+          req: {foo: "bar"}
+        };
+
+        return errorLoggerTestHelper(options).then(function (result) {
+          result.transportInvoked.should.eql(true);
+        });
+      });
+
+      it('should not log error when function returns true', function() {
+        var options = {
+          req: {foo: "bar"},
+          loggerOptions: {
+            skip: function() {return true;}
+          }
+        };
+
+        return errorLoggerTestHelper(options).then(function (result) {
+          result.transportInvoked.should.eql(false);
+        });
+      });
+
+      it('should throw an error if skip is not a function', function() {
+        var loggerFn = expressWinston.errorLogger.bind(expressWinston, {skip: 'not a function'});
+        loggerFn.should.throw();
```

Comment:
This is actually throwing due to there being no `transports` or `instance` provided to the error logger. If you change the `loggerFn` declaration to
```javascript
var loggerFn = expressWinston.errorLogger.bind(expressWinston, {
  skip: 'not a function',
  transports: [new MockTransport({})]
});
```
the test will fail.

Reply:
I removed the test